docs/introduction/faq.md

```diff
@@ -0,0 +1,95 @@
+
+For tokens outside the Cosmos system, they can only be introduced via pegged
+derivatives. Read about interoperating with existing blockchains here: [Peggy](https://blog.cosmos.network/the-internet-of-blockchains-how-cosmos-does-interoperability-starting-with-the-ethereum-peg-zone-8744d4d2bc3f).
+
+``` 
+           _ peg smart contract
+          /
+  [  Ethereum  ] <--> [ EtherCosmos Peg Zone ] <-IBC-> [  Cosmos Hub  ] <-IBC-> (Bitcoin) [ PoW/Casper ]
+                      [      Tendermint      ]         [  Tendermint  ] <-IBC-> (exchange)
+```
+
+### How does Cosmos manage governance?
+
+In Cosmos, [the stakeholders are well defined](/validators/overview.md), as is the prior social contract. Ethereum had a hard time with the fork because they had to ask the ether holders as well as the miners, but the ether holders had no prior social contract or obligation to partake in governance, so no quorum could be reached in time. Asking the miners is necessary to ensure that the hard-fork will have support, but after a while they tend to simply follow the money and incentives.
+
+Cosmos is different because instead of anonymous miners we have social contract bound validators and delegators who have stake, and, they have the obligation to partake in governance.
+
+## Validators
+
+### What is the maximum number of validators in Cosmos?  What about nodes?
+
+We will start with [100 validators](/validators/overview.md). Anyone else can be a node. To start, the validators will be the same across all shards - they will run the shards concurrently. Over time, these restrictions will be loosened. Misbehaviour in the consensus on any shard will result in security deposits being revoked.
+
+### What will be the process for abandoning validators that misbehave?
+
+If a validator misbehaves on its own by double-signing at the same height &amp; round, then the evidence is very short and simple -- it's just the two conflicting votes.  This evidence can be included in the the Cosmos Hub as a Slash transaction, and the validator will immediately become inactive and slashed after the Slash transaction gets committed.
+
+If there is a zone fork, either of the Cosmos Hub or any of the zones, the two conflicting commits also constitute evidence.  This is a much more complicated data structure.  It is guaranteed to slash at least 1/3 of the validators' atoms for that zone.
+
+### What's the difference between a Delegator and a Validator?
+
+A [validator](/validators/overview.md) has an active key involved in signing votes in the consensus protocol. A validator must also have some Atoms in a security deposit. Since there will only be a limited number of validators, [other Atom holders can delegate](/resources/delegator-faq.md) to the validators, thereby contributing to the economic security of the system by putting their funds on the line if the validator misbehaves. In return, they earn a share of the transaction fees and any inflationary rewards.
```

Comment:
Should we add a line saying delegators must choose validators wisely because they also get slashed if validator get slashed?